src/domains/wallet/components/ReceiveFunds/ReceiveFundsForm.tsx

```diff
@@ -19,6 +19,11 @@ export const ReceiveFundsForm = () => {
 		register("amount");
 	}, [register]);
 
+	const isSmartbridgeUsedInNetwork = useMemo(() => {
+		const smartbridgeNetworks = ["ark.mainnet", "ark.devnet"];
```

Comment:
this should actually depend on the network configuration, e.g. https://dwallets.ark.io/api/node/configuration will show a value for the smartbridge that determines if they are enabled or not. It shouldn't be hardcoded to just ark mainnet/devnet as there can be plenty of custom networks that also support smartbridge

Reply:
Thanks @ItsANameToo. Is this network configuration available for the other supported networks with a similar structure?

Reply:
PSDK will handle it as brian mentioned, not something that should concern the wallet. It will be similar for all ARK based networks, but other networks will need special handling as they might not have it at all or in a different fashion

Reply:
Great :+1:

Reply:
@faustbrian @ItsANameToo patched the fix using `network?.can(FeatureFlag.MiscellaneousMemo);`